.gitignore

```diff
@@ -1 +1,5 @@
 node_modules/
+jmeter.log
```

Comment:
They get added if you want to update the codebase and submit a PR unless you include them in .gitignore.